modules/databricks-loader/src/main/resources/application.conf

```diff
@@ -1,5 +1,6 @@
 "snowplow": {
     "storage": {
         "type": "databricks"
+        "catalog": "hive_metastore"
```

Comment:
Is this a right default?

Reply:
Yes, in Databricks if you don't specify a catalog then it uses `hive_metastore` by default.  Here we are making that default explicit.  Which works nicely, because it also means user can override it with something different.